Ago-Dic-2020/escarcega-ramirez-erick-orlando/Practicas/Practica1/practica1.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+
+class Matrix:
+    def __init__(self, matrix_string):
+       self.matrix = [[int(a) for a in line.split(' ')] for line in matrix_string.splitlines()]
+
+    def row(self, row):
+        return self.matrix[row-1]
+
+    def column(self, col):
+        return [self.matrix[k][col-1] for k in range(len(self.matrix))]
```

Comment:
¿De qué otra manera podríamos llevar a cabo esto sin utilizar la función `range`?

Reply:
return [k[col-1] for k in self.matrix]
indexando el row que nos da el for:  sacando las columnas desde el row que regresa el for, osease un for each
en donde k es la de lista de lista de listas y el col-1 es la posicion de esa lista que se traera

Reply:
Muy bien!